src/pipecat/services/gemini_multimodal_live/gemini.py

```diff
@@ -492,13 +515,20 @@ def __init__(
         params: Optional[InputParams] = None,
         inference_on_context_initialization: bool = True,
         file_api_base_url: str = "https://generativelanguage.googleapis.com/v1beta/files",
+        http_options: Optional[HttpOptions] = None,
         **kwargs,
     ):
         """Initialize the Gemini Multimodal Live LLM service.
 
         Args:
             api_key: Google AI API key for authentication.
             base_url: API endpoint base URL. Defaults to the official Gemini Live endpoint.
+
+                .. deprecated:: 0.0.88
+                    This parameter is deprecated and no longer has any effect.
+                    Please use `http_options` to customize requests made by the
```

Comment:
See rationale in [this commit message](https://github.com/pipecat-ai/pipecat/pull/2776/commits/f8b6587773d517a3a693ecdd26c355498524d29c). Note that `http_options` is a new argument here.